app/javascript/components/molecules/FormModalLoading.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Loading from '../molecules/Loading'
+
+const FormModalLoading = () => {
```

Comment:
Just adding a note that we might need a loading HOC to handle all the loadings for components. Nothing we have to implement now. Just making a note.